server.js

```diff
@@ -66,6 +67,20 @@ app.put('/api/foods/:id', function(request, response) {
   })
 })
 
+app.get('/api/diary', function(request, response) {
```

Comment:
@case-eee we are working on getting the index for the food log, and we had imagined our JSON would return the following:
```
{ id: 1,
  name: 'Breakfast',
  foods:
   [ anonymous {
       id: 1,
       name: 'Banana',
       calories: 105,
       created_at: 2017-07-02T21:28:15.563Z,
       food_id: 1,
       meal_id: 1 },
     anonymous {
       id: 2,
       name: 'French Silk Pie',
       calories: 340,
       created_at: 2017-07-02T21:28:15.569Z,
       food_id: 2,
       meal_id: 1 } ] }
```
We tried many approaches, and currently we are able to create the above but unable to return it outside the callback methods - perhaps having to do with the asynchronous nature of javascript?